### PR TITLE
Hw 2

### PR DESCRIPTION
**Pull request by denys-buhlak**

### CODE REVIEW

File: hw-2/src/html/layouts/header/_banner.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<section class="banner">
+    <div class="container">
+        <div class="banner-wrap">
```
Comment (marynakorol) [inline hw-2/src/html/layouts/header/_banner.html:3]: text and button should be in one line on desktop
image

File: hw-2/src/scss/components/_banner.scss [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    }
+
+    &-wrap {
+        display: flex;
+        flex-wrap: wrap;
+        justify-content: center;
+        align-items: center;
+    }
+
+    &-title {
```
Comment (marynakorol) [inline hw-2/src/scss/components/_banner.scss:22]: title shouldn't be bold

File: hw-2/src/html/layouts/_main.html [hunk 3]
```diff
@@ -0,0 +1,54 @@
+<main>
```
Comment (marynakorol) [inline hw-2/src/html/layouts/_main.html:1]: image
buttons position is wrong

File: hw-2/src/html/layouts/_main.html [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    <!-- <div class="test"> <p>Lorem ipsum dolor sit amet consectetur adipisicing
```
Comment (marynakorol) [inline hw-2/src/html/layouts/_main.html:2]: tooltip missed
image

File: hw-2/src/html/layouts/_header.html [hunk 5]
```diff
@@ -0,0 +1,11 @@
+<header class="site-header">
+    <div class="site-header-wrap container is-container-wide">
+        <a class="header-logo"
+           href="./">SKOUT</a>
+        <button class="burger">
```
Comment (marynakorol) [inline hw-2/src/html/layouts/_header.html:5]: the button is enormous when clicked (during transition) remove this behavior, please
image

File: hw-2/src/scss/layouts/_footer.scss [hunk 6]
```diff
@@ -0,0 +1,167 @@
...
+
+    list-style-type: none;
+}
+
+.footer-social-item {
+    margin: 10px;
+    padding: 10px;
+}
+
+.footer-social-link {
```
Comment (marynakorol) [inline hw-2/src/scss/layouts/_footer.scss:156]: wrong position on mobile 
image

File: hw-2/src/scss/utils/_break-points.scss [hunk 7]
```diff
@@ -0,0 +1,31 @@
+@mixin respond-to($breakpoint) {
+    @media (min-width: $breakpoint) {
+        @content;
+    }
+}
+
+// $breakpoints: (
```
Comment (marynakorol) [inline hw-2/src/scss/utils/_break-points.scss:7]: useless comments

**Review by marynakorol — CHANGES_REQUESTED** [review]
needs to add sliders and change some styles

File: hw-2/src/scss/utils/_break-points.scss (hunk 7 above)
Comment (denys-buhlak) [reply hw-2/src/scss/utils/_break-points.scss:7]: fixed

File: hw-2/src/js/parts/_manageMenu.js [hunk 8]
```diff
@@ -0,0 +1,36 @@
+export const manageMenu = () => {
+    const burgerButton = document.querySelector('.burger');
+    const navBlock = document.querySelector('.main-nav');
+    const pageNavLinks = document.querySelectorAll('.pages-nav-link');
+    
+    burgerButton.addEventListener('click', () => {
+        if(window.innerWidth < 1024){
```
[the remark this answers is not in the document]
Comment (denys-buhlak) [reply hw-2/src/js/parts/_manageMenu.js]: fixed

File: hw-2/src/scss/base/_common.scss [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+}
+
+body {
+    margin: 0;
+    padding: 0;
+
+    line-height: 24px;
+    font-weight: normal;
+    font-size: 16px;
+    font-family: $ff--primary;
```
[the remark this answers is not in the document]
Comment (denys-buhlak) [reply hw-2/src/scss/base/_common.scss]: fixed

File: hw-2/src/scss/components/_banner.scss [hunk 10]
```diff
@@ -0,0 +1,76 @@
...
+        margin: 20px 0;
+        padding: 0 20px;
+
+        line-height: 24px;
+        font-size: 13px;
+        font-family: $ff--primary;
+
+        @include respond-to($bp--tablet) {
+            line-height: 25px;
+            font-size: 16px;
```
[the remark this answers is not in the document]
Comment (denys-buhlak) [reply hw-2/src/scss/components/_banner.scss]: fixed

File: hw-2/src/scss/base/_fonts.scss [hunk 11]
```diff
@@ -0,0 +1,16 @@
+// EXAMPLE (in *.scss)
```
[the remark this answers is not in the document]
Comment (denys-buhlak) [reply hw-2/src/scss/base/_fonts.scss]: fixed

File: hw-2/src/js/parts/_manageMenu.js [hunk 12]
```diff
@@ -0,0 +1,36 @@
...
+            navBlock.classList.toggle('is-main-nav-visible');
```
[the remark this answers is not in the document]
Comment (denys-buhlak) [reply hw-2/src/js/parts/_manageMenu.js]: fixed

File: hw-2/src/html/layouts/_main.html (hunk 3 above)
Comment (denys-buhlak) [reply hw-2/src/html/layouts/_main.html]: fixed

File: hw-2/src/html/layouts/_main.html (hunk 4 above)
Comment (denys-buhlak) [reply hw-2/src/html/layouts/_main.html]: fixed
Not sure that good way

File: hw-2/src/scss/layouts/_footer.scss (hunk 6 above)
Comment (denys-buhlak) [reply hw-2/src/scss/layouts/_footer.scss]: Not fixed.
I use another layouts (discuss by phone)

File: hw-2/src/html/layouts/_header.html (hunk 5 above)
Comment (denys-buhlak) [reply hw-2/src/html/layouts/_header.html:5]: fixed

File: hw-2/src/scss/components/_header-logo.scss [hunk 13]
```diff
@@ -0,0 +1,34 @@
+.header-logo {
+    display: block;
+    width: 80px;
+    height: auto;
+
+    font-size: 0;
+
+    // background-image: url('../../images/header-logo-light.svg');
```
[the remark this answers is not in the document]
Comment (denys-buhlak) [reply hw-2/src/scss/components/_header-logo.scss]: fixed

File: hw-2/src/html/layouts/header/_banner.html (hunk 1 above)
Comment (denys-buhlak) [reply hw-2/src/html/layouts/header/_banner.html:3]: fixed

File: hw-2/src/scss/components/_banner.scss (hunk 2 above)
Comment (denys-buhlak) [reply hw-2/src/scss/components/_banner.scss]: fixed

File: hw-2/src/scss/components/_header-logo.scss [hunk 14]
```diff
@@ -0,0 +1,34 @@
...
+    background-repeat: no-repeat;
+    background-position: center center;
+
+    transition-timing-function: $trs--type;
+    transition-duration: $trs--time--long;
+    transition-property: transform, outline, opacity;
+
+    @include respond-to($bp--desktop) {
+        flex-shrink: 0;
+        width: 110px;
```
[the remark this answers is not in the document]
Comment (denys-buhlak) [reply hw-2/src/scss/components/_header-logo.scss]: fixed

File: hw-2/src/scss/components/_burger.scss [hunk 15]
```diff
@@ -0,0 +1,87 @@
...
+        content: '';
+        position: absolute;
+        left: 0;
+
+        width: 100%;
+        height: 100%;
+
+        background-color: inherit;
+
+        transition: all 1s ease;
```
[the remark this answers is not in the document]
Comment (denys-buhlak) [reply hw-2/src/scss/components/_burger.scss]: fixed

File: hw-2/src/scss/layouts/footer/_social-link.scss [hunk 16]
```diff
@@ -0,0 +1,80 @@
...
+    }
+}
+
+.icon {
+    width: 50px;
+    height: 50px;
+
+    background-repeat: no-repeat;
+    background-position: center center;
+    background-size: 200%;
```
Comment (marynakorol) [inline hw-2/src/scss/layouts/footer/_social-link.scss:63]: better to use contain prop here

**Review by marynakorol — APPROVED** [review]

**Pull request merged**